packages/lesswrong/components/common/ForumDropdownMultiselect.tsx

```diff
@@ -0,0 +1,167 @@
+  root: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    color: theme.palette.grey[600],
+    textAlign: "center",
+    ...(!isEAForum && {
+      "& button:hover": {
+        backgroundColor: "transparent",
+      },
+    }),
+  },
+  button: {
+    textTransform: "none",
+    boxShadow: "none",
+    padding: 0,
+    fontSize: "14px",
+    minHeight: 32,
+    minWidth: 'unset',
+    cursor: "pointer",
+    color: theme.palette.primary.main,
+    paddingLeft: 12,
+    paddingRight: 6,
+    backgroundColor: "transparent",
+    ...(isEAForum && {
+      color: "inherit",
+      "&:hover": {
+        backgroundColor: theme.palette.grey[250],
+        color: theme.palette.grey[1000],
+      },
+    }),
+  },
+  openButton: {
+    ...(isEAForum && {
+      backgroundColor: theme.palette.grey[250],
+      color: theme.palette.grey[1000],
+    }),
+  },
+  dropdownIcon: {
+    verticalAlign: "middle",
+    position: "relative",
+    ...(isEAForum && { width: 16, height: 16, marginLeft: 4, padding: 1}),
+  },
+  padding: {
+    width: 10,
+  },
+  selectedIcon: {
+    verticalAlign: "middle",
+    position: "relative",
+    color: theme.palette.primary.main,
+    marginLeft: "auto",
+    marginRight: 2,
+    width: 19,
+    height: 19,
+  },
+  menu: {
+    marginTop: 28,
+    ...(isEAForum && {
+      "& a:hover": {
+        opacity: "inherit",
+      },
+    }),
+    '& .MuiList-padding': {
+      padding: '4px 0px',
+    }
+  },
+  menuItem: {
+    ...(isEAForum && {
+      color: theme.palette.grey[1000],
+      borderRadius: theme.borderRadius.small,
+      padding: "6px 8px",
+      margin: "0px 4px",
+      fontSize: "14px",
+      lineHeight: "14px",
+      "&:hover": {
+        backgroundColor: theme.palette.grey[250],
+        color: theme.palette.grey[1000],
+      },
+    }),
+    "&:focus": {
+      outline: "none",
+    },
+  },
+});
+
+const ForumDropdownMultiselect = ({values, options, queryParam, onSelect, eventProps, classes, className}:{
+  values: string[],
+  options: Record<string, SettingsOption>,
+  queryParam?: string,
+  onSelect?: (value: string) => void,
+  eventProps?: Record<string, string>
+  classes: ClassesType,
+  className?: string,
+}) => {
+  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null)
+  const { captureEvent } = useTracking()
+  const label = values.reduce((prev, next) => {
+    const nextLabel = options[next].shortLabel || options[next].label
+    if (!prev) return nextLabel
+    return `${prev}, ${nextLabel}`
+  }, '')
+  const { MenuItem, ForumIcon } = Components;
+
+  const dropdownIcon = isEAForum ? <ForumIcon icon="ThickChevronDown" className={classes.dropdownIcon} /> : <ArrowDropDownIcon className={classes.dropdownIcon}/>
+  return (
+    <div className={classNames(classes.root, className)}>
+      <Button
+        variant="contained"
+        onClick={(e) => {
+          captureEvent("ForumDropdownMultiselectOpened", {value: values, ...eventProps})
```

Comment:
This used to be `captureEvent("forumDropdownOpened", {value, ...eventProps})`

I actually don't know why we're tracking this - do you think it's useful? I kind of want to just remove it now, rather than properly adapt it for the multiselect

Reply:
@Will-Howard I'm going to remove these two `captureEvent`s from the file then, unless you are using them for something?